modules/storage-api/src/testFixtures/java/org/apache/ignite/internal/storage/impl/TestMvPartitionStorage.java

```diff
@@ -586,6 +586,10 @@ void startRebalance() {
     void abortRebalance() {
         checkStorageClosed();
 
+        if (!rebalanced) {
```

Comment:
Name is confusing. Past simple implies that rebalance already happened, not "in progress"